index.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+const winston = require('winston');
+
+const logger = winston.createLogger({
+    level: process.env.LOG_LEVEL || 'info',
+    transports: [
+        new (winston.transports.Console)({ timestamp: true })
+    ]
+});
+
```

Comment:
Can we have the logging wrapper with functions logInfo, logError etc and have a standard logging pattern in json format. Pattern can also have metadata of the log including job or build id where ever applicable and a unique correlation id which can be passed across components like api to queue worker etc.

Reply:
https://github.com/screwdriver-cd/logger/pull/2/files now adds json logging. `winston` does support overriding log format. https://github.com/winstonjs/winston#formats we can define someinfor globally and then refer to it in the `printf` format